### PR TITLE
Add navigation bar into webpage

### PR DESCRIPTION
**Pull request by hienphan0111**
1. Modify the Awesome books application to have:

- [ ] A Navigation bar.
- [ ] Three content sections:

- Books list.
- Add book form.
- Contact info.

2. Use this wireframe as a guide:

### CODE REVIEW

**Review by ishimwezachee — COMMENTED** [review]
Hi  @hienphan0111 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
image


## Invalid Code Review Request

You have submitted a project, that was supposed to have two contributors, because it is a pair programming project.  Let your partner-- contribute to the project as well so that it can be reviewed.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: script.js [hunk 1]
```diff
@@ -77,3 +85,25 @@ add.addEventListener('click', () => {
...
+addMn.addEventListener('click', () => {
+  allBooks.classList.add('hidden');
+  addBook.classList.remove('hidden');
+  contact.classList.add('hidden');
+});
+contactMn.addEventListener('click', () => {
+  allBooks.classList.add('hidden');
+  addBook.classList.add('hidden');
+  contact.classList.remove('hidden');
+});
```
Comment (OyePriscilla) [inline script.js:109]: - [ ] You've both done a great job I must say, but I noticed that the `hidden `class is not effective for the three sections. ℹ️ 

## Your Project
png-transparent-nigerian-naira-currency-symbol-symbol-miscellaneous-angle-text

File: index.html [hunk 2]
```diff
@@ -9,21 +9,44 @@
...
+      <ul calss="info">
+        <li>Our e-mail: mail@mail.com</li>
+        <li>Our phone number: 004358634422</li>
+        <li>Our address: Streetname 22, 84503 City, Country</li>
+      </ul>
+    </section>
   </main>
+  <footer>
+    <p>Copyright ...</p>
+  </footer>
```
Comment (OyePriscilla) [inline index.html:49]: - [ ] Please ensure that the footer is fixed at the bottom of the page  for good user experience. 👍

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @hienphan0111  and @Nezerwa 😃, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-so-close-seth-meyers

## Highlights
Github flow observed✔️
Linters Set up correctly ✔️ 

## Required Changes ♻️
**Kindly check the inline comments below. Thanks. 👍** 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codecaiine — APPROVED** [review]
Hi @hienphan0111 @Nezerwa  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Great project implementation :heavy_check_mark: 
- Good use Github flow :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**